bundle/manifests/tackle.konveyor.io_addons.yaml

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
@jortel could you remove this space, is failing to validate bundle as operator-sdk it does not understand it

Reply:
Fixed.